Cargo.toml

```diff
@@ -24,10 +24,10 @@ secp-recovery = ["secp256k1/recovery"]
 [dependencies]
 bech32 = "0.7.2"
 bitcoin_hashes = "0.9.1"
-secp256k1 = "0.20.0"
+secp256k1 = "0.20.1"
```

Comment:
This seems very unrelated in this PR/commit.

Reply:
Removed. It was not an oversight, but just me committing into the wrong branch (version update is required for error derives, which is parallel PR)